source/_statement.html.md

```diff
@@ -1,5 +1,5 @@
 # Accessibility statement for GovWifi
-This service is run by the Government Digital Service (GDS). We want as many people as possible to be able to use it. For example, that means you should be able to:
+This service is run by the Cabinet Office CDIO. We want as many people as possible to be able to use it. For example, that means you should be able to:
```

Comment:
Remove CDIO, it doesn't work grammatically and just saying Cabinet Office is fine.

Reply:
Gone with the latter... further discussion welcome :)

Reply:
Should the brackets be reversed?